omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
If you read the ticket description, what needs to be done is very clear, annotating a controller mapped to /admin doesn't work because the URL patterns mapped to the spring dispatcher servlet don't match URLs like /admin. a URL needs to end with in a certain form like <b>.list</b>, <b>.htm</b>, <b>.form</b> etc to be processed by the dispatcher servlett. The solution is to just change all these existing URLs to end with <b>index.htm</b>